plugins/recorded_future/unit_test/util.py

```diff
@@ -15,7 +15,7 @@ class Util:
     @staticmethod
     def default_connector(action: insightconnect_plugin_runtime.Action, params=None):
         if not params:
-            params = {Input.API_KEY: {"secretKey": "9de5069c5afe602b2ea0a04b66beb2c0"}}
+            params = {Input.API_KEY: {"secretKey": "***REMOVED***"}}
```

Comment:
Please ignore the return of the Recorded Future API key. We will have to use BFG again, in a separate ticket, before we rebase this PR.